### DISCUSSION TITLE
Is there any way to store array in sqllite model?

### DISCUSSION THREAD

**Discussion by indahud** [prisma/prisma#5112] [Q&A] [answered]
I have created a model. I want to store an array.
`model Names {
  id Int @id @default(autoincrement())
  data String[]
}`
It gives me a error `Field in model cant be list. Current connector dos not support list of primitive types`
I even tried defining it like this `data [String!]!`

What is the way to define an array here.

**Comment by ryands17** [comment, marked as answer]
Hey @indahud 👋 

Currently Prisma doesn't support creating a list of scalars in SQLite as that isn't supported by the database itself. I would suggest creating a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this.

If you want to use a list of Strings, you would need to go for PostgreSQL or MySQL currently.

**Reply by Bashsoft707** [reply]
I'm having the same issue using mysql

**Reply by soymartinez** [reply]
I am using MySQL and I have the same problem with a list of strings.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
